python/test/test_api.py

```diff
@@ -418,6 +420,9 @@ def test_ready(self, server_options):
         server = tritonserver.Server(server_options).start()
         assert server.ready()
 
+    @pytest.mark.skip(
+        reason="Skipping test, some request/response object may not be released which may cause server stop to fail"
```

Comment:
Can we use xfail instead of skip?

@rmccorm4 - what do you think - as we had spent time on fixing this - how much an issue is this?

Reply:
yes, switched to xfail.